cmd/init.go

```diff
@@ -0,0 +1,103 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/spf13/cobra"
+)
+
+var yesUseDefaults bool
+
+// initCmd represents the init command
+var initCmd = &cobra.Command{
+	Use:   "init",
+	Short: "Interactively create a harbor-compose.yml file",
+	Long: `This will ask you a bunch of questions, and then write a harbor-compose.yml for you.
+
+If you invoke it with -y or --yes it will use only defaults and not prompt you for any options.`,
+	Run: initHarborCompose,
+}
+
+func init() {
+	initCmd.PersistentFlags().BoolVarP(&yesUseDefaults, "yes", "y", false, "don't ask questions and use defaults")
+	RootCmd.AddCommand(initCmd)
+}
+
+func initHarborCompose(cmd *cobra.Command, args []string) {
+
+	name := "mss-harbor-compose-app"
+	env := "dev"
+	barge := "digital-sandbox"
+	replicas := "2"
+	group := "mss"
+	property := "turner.com"
+	project := "poc"
+	product := "poc"
+
+	//ask questions
+	if !yesUseDefaults {
+		name = promptAndGetResponse("shipment name: (e.g., mss-my-app) ")
+		env = promptAndGetResponse("shipment environment: (dev, qa, prod, etc.) ")
+		barge = promptAndGetResponse("barge: (cnn, nba, digital-sandbox, corp-sandbox, etc.) ")
```

Comment:
can you actually list all the available options here for now. At least the user will get some insight into what they can choose.